config/install/acquia_lift.settings.yml

```diff
@@ -17,4 +17,5 @@ thumbnail: []
 visibility:
   # Track, only when the page URL falls OUTSIDE OF these path patterns.
   path_patterns: "/admin\n/admin/*\n/batch\n/node/add*\n/node/*/*\n/user/*/*\n/block/*"
+content_replacement_mode: 'untrusted'
```

Comment:
1. The reason I feel "deployment" fits users' anticipation better than "replacement", is because when I was observing users, the impression I have perceived was they tend to think and ask "how do I put content here", and not what I as a dev would usually think "how do I replace this element by a content element". In other words "replacement" does not fit my user mentality. @tottey do you share the same thought or am I overthinking it? 

2. For me, this "extra" and/or "noise" isn't actually a bad thing, since it provides additional semantic value, and describes the nature of this parameter more accurately, leaving no ambiguity for the next dev. Plus, it is always a bonus for consistency, even just between the text and underlying parameter name. Lastly, there is no notable penalty for longer names, but redundant ones.

On the other hand, I am interested in showing you what I think for you consideration, and not trying to enforce - I am sure neither of us is terribly wrong, and if we are debating I'd prefer moving the story forward without change.